opentsdb-ingestion.md

```diff
@@ -0,0 +1,160 @@
+## OpenTSDB Ingestion
+
+There are 2 methods for ingesting OpenTSDB data into IRONdb:
+
+1. RESTful HTTP POST of OpenTSDB JSON formatted datapoint(s)
+2. Network socket listener akin to the normal OpenTSDB telnet method
+
+For the HTTP method, POST a JSON object (or an array of JSON objects) to
+the RESTful API endpoint (see the section below - *Writing OpenTSDB with HTTP*).
+Each datapoint should be encoded as follows:
+
+```
+{
+  "metric": "metric_name",
+  "timestamp": timestamp,
+  "value": value,
+  "tags": {
+    "tag_key": "tag_value",
+    "tag_key2": "tag_value2",
+    ...
+    "tag_keyn": "tag_valuen"
+  }  
+}
+```
+
+At least one tag key/value pair is required. Multiple datapoints can be sent
+as a JSON array of the datapoints, separated by commas and the entire POST
+enclosed in square brackets.
+
+For example:
+
+```
+[{
+  "metric": "my.metric.name",
+  "timestamp": 1544678300,
+  "value": 637,
+  "tags": {
+    "datacenter": "east"
+  }
+},
+{
+  "metric": "myother.metric.name",
+  "timestamp": 1544688100,
+  "value": 3475,
+  "tags": {
+    "datacenter": "west"
+  }
+}]
+```
+
+In the case of the telnet method, telnet `put` commands in the normal OpenTSDB
+format are accepted:
+
+`put<space>metric_name<space>timestamp<space>value<space>tag_key=tag_value{<space>tag_key=tag_value...}`
```

Comment:
Shouldn't there be a separator of some sort between the tag_value, and the ...?

Reply:
the separator is space before/between each tag=value and you don't want spaces elsewhere - the curly brackets show that you can optionally have more tag=value (I updated it a little bit to be clearer).  you have to have at least 1 tag=value.